draft-bbs-signatures.md

```diff
@@ -346,18 +346,27 @@ Procedure:
 
 Sign computes a signature from SK, PK, over a vector of messages. This method
 describes deterministic signing. For threshold signing, XOF can be replaced
-with a PRF due to the insecurity of deterministic threshold signing.
+with a PRF due to the insecurity of deterministic threshold signing. 
+
+The method takes as an input a list of the generators (H_s, H_d, H_1, ..., H_L). The first generator from that list (H_s), is used for the blinding value (s) of the signature. The second generator (H_d), is used to sign the signature's domain separation tag (sig_dst), which binds both signature and proof to a specific domain. Finally, the rest of the generators (H_1, ..., H_L), are used for the signed messages.
 
 ```
-signature = Sign(SK, PK, (msg_1,..., msg_L), (H_1,..., H_L))
+signature = Sign(SK, PK, (msg_1,..., msg_L), (H_s, H_d, H_1, ..., H_L))
 
 Inputs:
 
 - msg_1,...,msg_L, octet strings. Messages to be signed.
-- H_1,..., H_L, points of G1. Generators used to sign the messages.
+- H_s, H_d, H_1,..., H_L, points of G1. Generators used to create the signature. H_s is
+                          used for the signature blinding value, H_d for the signature 
+                          domain separation tag and the rest for the signed messages.
 - SK, a secret key output from KeyGen
 - PK, a public key output from SkToPk
 
+Parameters:
+
+- CipherInfo, an optional string containing ciphersuite specific information.
```

Comment:
What makes cipherinfo a parameter and not an input here?

Reply:
My thinking was that if cipherinfo is optional (which i think it should), that can be cached for different calls of the operations using the same ciphersuite, IMO it is suited for a parameter. This way, the inputs are only the values necessary for the core "mathematical" operations (in contrast, cipherinfo is only used for the sig_dst). This is similar to what `keyGen` does with `key_info`.

Reply:
Ok understood and is consistent as you said with keyGen